### PR TITLE
Move layer manager to the end of examples (from Paths to WMTS)

### PR DESCRIPTION
**Pull request by Beak-man**
### Description of the Change
Moved every creation of the LayerManager to the end of each example in order to increase readability. In some examples, this statement was in the middle of them, disrupting the "narrative" of the example.

### Applicable Issues
Related to #725

### CODE REVIEW

File: examples/StarField.js [hunk 1]
```diff
@@ -76,4 +75,7 @@ requirejs([
             }
         }
 
+        // Create a layer manager for controlling layer visibility.
+        var layerManager = new LayerManager(wwd);
+
```
Comment (eirizarry) [inline examples/StarField.js]: Extra line here.

File: examples/TestMovingSurfaceShapes.js [hunk 2]
```diff
@@ -254,5 +251,8 @@ requirejs(['./WorldWindShim',
             return rotatingSectors[surfaceSector.sectorCounter];
         }
 
+        // Create a layer manager for controlling layer visibility.
+        var layerManager = new LayerManager(wwd);
+
```
Comment (eirizarry) [inline examples/TestMovingSurfaceShapes.js]: Extra line and move bracket to the same line as parenthesis and semicolon.

File: examples/WMS.js [hunk 3]
```diff
@@ -82,4 +79,7 @@ requirejs(['./WorldWindShim',
 
         $.get(serviceAddress).done(createLayer).fail(logError);
 
+        // Create a layer manager for controlling layer visibility.
+        var layerManager = new LayerManager(wwd);
+
```
Comment (eirizarry) [inline examples/WMS.js]: Extra line here.

**Comment by pdavidc** [comment]
@Beak-man @eirizarry Have you been applying the project code formatting to these changes?

**Comment by Beak-man** [comment]
@pdavidc I overlooked using the code formatting tool. I'll update the PR.

**Pull request merged**